api/serializers.py

```diff
@@ -20,6 +27,11 @@ class Meta:
                   'week_assignments_capacity', 'wanted_assignments', 'phone_number', 'email', 'parental_consent',
                   'languages']
 
+    def validate(self, data):
+        parental_consent_validator(data)
```

Comment:
Does DRF capture the ValidationError thrown here and presents the error message in the JSON? Or does the exception actually raise? (asking to validate)

Reply:
DRF catches the exception and returns the message in non_field_error json field as a response.
![image](https://user-images.githubusercontent.com/33120303/79377759-db3ebb00-7f64-11ea-9432-cfc87d72c676.png)

Reply:
Nice.